test/gtest/uct/ib/test_dc.cc

```diff
@@ -517,6 +539,8 @@ UCS_TEST_P(test_dc_flow_control, general_disabled)
 
 UCS_TEST_P(test_dc_flow_control, pending_grant)
 {
+    allow_misordering misorder(m_e2);
```

Comment:
pls comment in each test why waitq reorder happens

Reply:
> pls comment in each test why waitq reorder happens

added comment